avocado/core/runners/utils/messages.py

```diff
@@ -215,3 +216,13 @@ def start_logging(config, queue):
 
     sys.stdout = StreamToQueue(queue, "stdout")
     sys.stderr = StreamToQueue(queue, "stderr")
+
+    # store custom test loggers
+    enabled_loggers = config.get('job.run.store_logging_stream') or ['avocado.test.progress']
```

Comment:
Are you proposing that `avocado.test.progress` becomes a special logging stream?  This is not what was implied from the current implementation.

Reply:
oh, this is a mistake :man_facepalming: , I was using that for some testing, I will remove that.